requirements.txt

```diff
@@ -0,0 +1,61 @@
+ale-py==0.7.4
```

Comment:
Let's delete or gitignore this. There should be one source of truth for dependencies and that is `pyproject.toml`.